pycti/entities/indicator/opencti_indicator_properties.py

```diff
@@ -0,0 +1,256 @@
+INDICATOR_PROPERTIES = """
```

Comment:
For code clarity, we want to extract the entities properties (used in GQL queries) in dedicated files.
This will 1) make the business code lighter, and 2) allow to use them in several files.

As the new `promote_to_indicator_v2` method returns an indicator, I'm already in 2) use-case.